README.md

```diff
@@ -86,12 +87,28 @@ source $HOME/pythonvirtenvs/capacity-status-env/bin/activate
 Now run the following to start the application in the root directory of the
 project:
 
+make project-start
 make api-start
 
-The application is configured to run on https://localhost:8443/apidoc/
+The application is configured to run on http://localhost:8000/apidoc/
```

Comment:
Might want to clearly specifiy that this is for development

Reply:
Done